markbook.py

```diff
@@ -6,47 +6,48 @@
 
 
 def create_assignment(name: str, due: str, points: int) -> Dict:
-    """Creates an assignment represented as a dictionary
-    
-    Args:
-        name: the name of the assignment.
-        due: the due date for the assignment.
-        points: what the assignment is out of (denominator).
-    Returns:
-        Assignment as a dictionary.
-    """
```

Comment:
Why remove the doc-string? You want to make the codebase even more difficult to read? LOL.